### PR TITLE
Update docs: troubleshooting with running postinstall

### PR DESCRIPTION
**Pull request by michellewong793**
Adding in the docs that if someone is trying to run the wallet and is running into package issues, try running the postinstall script will help!

### Description

Adds in a note for troubleshooting when setting up wallet for the first time locally. 

### Other changes

No other canges 

### Tested

Written and tested by miche on 1.14.2021 on iOS

### CODE REVIEW

File: packages/mobile/README.md [hunk 1]
```diff
@@ -417,6 +417,17 @@ To attach:
 
 ### Troubleshooting
 
+If you're having an error with installing packages, or `secrets.json` not existing: 
+
+try to run `yarn postinstall` in the monorepo `/celo-monorepo` after running `yarn install`. 
+
+A successful `yarn postinstall` looks like: [this](https://files.slack.com/files-pri/T9SEZ6M9B-F01JP5VDLJF/screen_shot_2021-01-13_at_2.39.56_pm.png).
```
Comment (jeanregisser) [inline packages/mobile/README.md]: This file is not public, you could add it to the repo or paste the text output instead.

File: packages/mobile/README.md [hunk 2]
```diff
@@ -417,6 +417,17 @@ To attach:
 
 ### Troubleshooting
 
+If you're having an error with installing packages, or `secrets.json` not existing: 
+
+try to run `yarn postinstall` in the monorepo `/celo-monorepo` after running `yarn install`.
```
Comment (jeanregisser) [inline packages/mobile/README.md]:
```suggestion
try to run `yarn postinstall` in the monorepo root folder after running `yarn install`. 
```

**Review by jeanregisser — COMMENTED** [review]
Great! 👍

File: packages/mobile/README.md (hunk 1 above)
Comment (michellewong793) [reply packages/mobile/README.md]: pasted text output

File: packages/mobile/README.md [hunk 3]
```diff
@@ -436,6 +436,57 @@ To attach:
 
 ### Troubleshooting
 
+If you're having an error with installing packages, or `secrets.json` not existing:
```
Comment (jeanregisser) [inline packages/mobile/README.md:451]: Could you add a title so it looks part of the troubleshooting items?

**Review by jeanregisser — APPROVED** [review]
great! 👍

**Pull request merged**